setup.py

```diff
@@ -46,7 +46,7 @@
       url = "http://open.eucalyptus.com",
       install_requires = ['paramiko >= 1.7','boto >= 2.5.2', 'jinja2 >= 2.7', 'argparse',
                           'futures', 'python-dateutil', 'mock', 'dnspython', 'pywinrm',
-                          'BeautifulSoup', 'requests >= 1', 'prettytable'],
+                          'BeautifulSoup', 'requests >= 1', 'prettytable', 'awacs'],
```

Comment:
Can we make this testcase without introducing a new dependency? Looks to me like this lib doesnt necessarily make anything easier than just having a JSON block quote and a format string. Additionally we are only using it in one place.